README.md

```diff
@@ -260,7 +260,6 @@ _CalDAV or CardDAV clients._
 
 ### Custom communication systems
 
-- [Broid](https://broid.ai) - Broid enables rich conversations on all messaging channels within a single schema integration using W3C standards. ([Demo](https://www.broid.ai), [Source Code](https://github.com/broidHQ/integrations)) `AGPL-3.0` `Nodejs`
```

Comment:
There hasn't been any commits to the repo since Feb 2018. Not old enough (IMO) to justify removing it alone, but the website is also down (replaced with a generic Heroku landing page). Together, yeah, it seems abandoned.

:+1: